pkg/nvidia/Dockerfile

```diff
@@ -0,0 +1,102 @@
+# syntax=docker/dockerfile-upstream:1.5.0-rc2-labs
+
+# Copyright (c) 2024 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+FROM lfedge/eve-alpine:82df60e43ab9f8c935584b8c7b4d0a4b0271d608 as build-base
+ENV BUILD_PKGS="autoconf automake build-base coreutils gettext git glib-dev go libtool libmd-dev ncurses-dev tar xz-dev yq zstd-dev"
+
+RUN eve-alpine-deploy.sh
+
+# This is a arm64 (only) package, nothing to do for other architectures
+FROM build-base as build-amd64
+FROM build-base as build-riscv64
+
+FROM build-base as build-arm64
+
+# Setup git to apply patches
+RUN git config --global user.email 'builder@projecteve.dev' && \
+    git config --global user.name 'Project EVE'
+
+# Jetson Linux tarball
+ENV JETSON_LINUX=https://developer.nvidia.com/downloads/embedded/l4t/r35_release_v5.0/release/jetson_linux_r35.5.0_aarch64.tbz2
+
+ADD ${JETSON_LINUX} /jetson_linux.tbz2
+
+# We need to build dpkg in order to get zstd compression support
+ENV DPKG_REVISION=1.22.6
+ADD --keep-git-dir=true https://salsa.debian.org/dpkg-team/dpkg.git#${DPKG_REVISION} /dpkg
```

Comment:
FWIW, AFAIK there is no way to get rid of this error, since it's not a rule, it's hadolint that doesn't recognize `--keep-git-dir` as a valid flag...

Reply:
You could do without `--keep-git-dir` and then before `git am`: `git init; git add .; git commit -m "initial commit"`

But IMO it is also fine to just add another hadolint-ignore line.

Reply:
I prefer to keep the `--keep-git-dir` line as we use in many other Dockerfile across pkg/. However, as I said, it's not a hadolint's rule, it doesn't recognize the flags, so unfortunately there is no `hadolint-ignore` to use...

Reply:
Fine.